src/dos/dos_locale.cpp

```diff
@@ -1627,6 +2131,26 @@ static const std::map<Country, CountryInfo> CountryData = {
 			Separator::Semicolon      // list separator
 		} }
 	} } },
+	{ Country::Syria, { "Syria", "SYR", {
+		{ LocalePeriod::Modern, {
+			// fr_SY
```

Comment:
I think this should be ar_SY
https://sourceware.org/git?p=glibc.git;a=blob;f=localedata/locales/ar_SY;h=015bd979adfa72cc01e3d832fe50e9d376c1407c;hb=HEAD